base/applications/cmdutils/more/more.c

```diff
@@ -500,8 +754,102 @@ int wmain(int argc, WCHAR* argv[])
         return 1;
     }
 
+    /* Parse flags */
+    HasFiles = FALSE;
+    for (i = 1; i < argc; i++)
+    {
+        if (argv[i][0] == L'/' || argv[i][0] == L'-')
+        {
+            switch (towupper(argv[i][1]))
+            {
+                case L'?':
+                    if (argv[i][2] == 0)
+                    {
+                        s_dwFlags |= FLAG_HELP;
+                        continue;
+                    }
+                    break;
+                case L'E':
+                    if (argv[i][2] == 0)
+                    {
+                        s_dwFlags |= FLAG_E;
+                        continue;
+                    }
+                    break;
+                case L'C':
+                    if (argv[i][2] == 0)
+                    {
+                        s_dwFlags |= FLAG_C;
+                        continue;
+                    }
+                    break;
+                case L'P':
+                    if (argv[i][2] == 0)
+                    {
+                        s_dwFlags |= FLAG_P;
+                        continue;
+                    }
+                    break;
+                case L'S':
+                    if (argv[i][2] == 0)
+                    {
+                        s_dwFlags |= FLAG_S;
+                        continue;
+                    }
+                    break;
+                case L'T':
+                    if (L'0' <= argv[i][2] && argv[i][2] <= L'9')
+                    {
+                        LPWSTR endptr;
+                        s_dwFlags |= FLAG_Tn;
+                        s_nTabWidth = wcstoul(&argv[i][2], &endptr, 10);
+                        if (*endptr == 0)
+                            continue;
+                    }
+                    break;
+                default:
+                    break;
+            }
+        }
+        else if (argv[i][0] == L'+')
+        {
+            if (L'0' <= argv[i][1] && argv[i][1] <= L'9')
+            {
+                LPWSTR endptr;
+                s_dwFlags |= FLAG_PLUSn;
+                s_nNextLineNo = wcstoul(&argv[i][1], &endptr, 10);
+                if (*endptr == 0)
+                    continue;
+            }
+        }
+
+        if (IsFlag(argv[i]))
+        {
+            ConResPrintf(StdErr, IDS_BAD_FLAG, argv[i]);
+            return 1;
+        }
+        else
+        {
+            HasFiles = TRUE;
+        }
+    }
+
+    if (s_dwFlags & FLAG_HELP)
+    {
+        ConResPuts(StdOut, IDS_USAGE);
+        return 0;
+    }
+
+    if (s_dwFlags & FLAG_C)
+    {
+        /* Clear the screen */
+        ConClearScreen(&Screen);
+    }
```

Comment:
This has to be done, not here, but **every time** a new page (not one line) is to be displayed.

Reply:
Moved.